test/prototype_common_utils.py

```diff
@@ -108,128 +298,217 @@ def randint_with_tensor_bounds(arg1, arg2=None, **kwargs):
     ).reshape(low.shape)
 
 
-def make_bounding_box(*, format, image_size=(32, 32), extra_dims=(), dtype=torch.int64):
+def make_bounding_box_loader(*, extra_dims=(), format, image_size=None, dtype=torch.float32):
```

Comment:
Why bounding box does not have `num_objects` arg ?

Reply:
Because we defined a bounding box as `(*, 4)`, i.e. `features.BoundingBox([0, 0, 10, 10], ...)` is valid although it only has a single dimension. Thus, if you want to have multiple boxes, set `extra_dims=(num_objects,)`